README.md

```diff
@@ -147,21 +139,20 @@ The boilerplate app comes with functional tests implemented with Ledger's [Ragge
 
 ### macOS / Windows
 
-To test your app on macOS or Windows, it is recommended to use the [VSCode integration](#with-vscode) to quickly setup a working test environment.
+To test your app on macOS or Windows, it is recommended to use [Ledger's VS Code extension](#with-vscode) to quickly setup a working test environment.
 
-You can use the following VsCode sequence of tasks (open the menu with `ctrl + shift + b` or `command + shift + b` on a Mac) :
+You can use the following sequence of tasks and commands (all accessible in the **extension sidebar menu**) :
 
-* `Run dev-tools image`,
-* `Build app` and select a device model.
+* `Select build target`
+* `Build app`
 
 Then you can choose to execute the functional tests :
 
-* `Install tests requirements` to be executed once after you have run the dev-tools image, followed by,
-* `Run functional tests` and select the model you just built the app for.
+* Use `Run tests`.
 
 Or simply run the app on the Speculos emulator :
 
-* `Test app with Speculos` and select the correct device model.
+* `Run with Speculos`.
 
 ### Linux (Ubuntu)
 
```

Comment:
This part should be updated "[VSCode integration](#with-vscode)" --> extension

Reply:
could be the good time to include this [PR](https://github.com/LedgerHQ/app-boilerplate/pull/90/files)

Reply:
Done. And I added the extension in `.vscoe/extensions.json`.